dll/win32/kernel32/client/thread.c

```diff
@@ -973,16 +972,48 @@ QueueUserAPC(IN PAPCFUNC pfnAPC,
 }
 
 /*
- * @implemented
+ * @unimplemented
```

Comment:
Well at least it reflects the **actual** state of the function, instead of claiming that it's implemented whereas it's not (it's just hackplemented or whatever you want to call it ^^).